package.json

```diff
@@ -1,13 +1,55 @@
 {
-  "name": "@salesforce/plugin-template",
-  "description": "A template repository for sfdx plugins",
-  "version": "1.0.0",
+  "name": "salesforcedx",
+  "version": "50.6.0",
+  "description": "",
   "author": "Salesforce",
   "bugs": "https://github.com/forcedotcom/cli/issues",
+  "files": [
+    "oclif.manifest.json",
+    "lib/**/*.js",
+    "commands/**/*.js",
+    "/lib",
+    "/messages",
+    "/oclif.manifest.json"
+  ],
+  "oclif": {
+    "commands": "./commands",
+    "plugins": [
+      "salesforce-alm",
+      "@salesforce/plugin-templates",
+      "@salesforce/sfdx-plugin-lwc-test",
+      "@salesforce/plugin-custom-metadata",
+      "@salesforce/plugin-apex",
+      "@salesforce/plugin-schema",
+      "@salesforce/plugin-user"
+    ]
+  },
+  "packagesToHardcodeVersions": [
```

Comment:
I'm not sure I love this name. Also, we only want the plugins to be autoupdated based on latest. I'm assuming that is what this is for?